java/api/src/main/java/io/ray/api/options/ActorCreationOptions.java

```diff
@@ -176,6 +180,16 @@ public Builder setMaxPendingCalls(int maxPendingCalls) {
       return this;
     }
 
+    /**
+     * Mark the creating actor as async.
```

Comment:
Better to add more descriptions. e.g,

If this flag doesn't match the Python code (e.g. the Python actor is/is not async but it's marked async/not async in Java), it will result in xxx errors.

Also, please specify the default value.